packages/devextreme/js/ui/chat.d.ts

```diff
@@ -108,6 +108,24 @@ export type Message = {
  * @docid
  */
 export interface dxChatOptions extends WidgetOptions<dxChat> {
+    /**
+     * @docid
+     * @default true
+     * @public
+     */
+    activeStateEnabled?: boolean;
```

Comment:
Please, add expectations into the defaultOption.tests.js

Reply:
Please pay attantion: https://github.com/DevExpress/DevExtreme/pull/27974/files#diff-58991390a814a09faf265a37cc2d68af413a6d61889acb16ae55b04734ad9308R1345